README.md

```diff
@@ -49,7 +53,9 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+town[:residents] = ["Maurice", "Gaston"]
+town[:castle][:guests] = "Belle"
```

Comment:
On this line you're setting `guests` to the string "Belle". Can you think of a way to do this that preserves the `guests` array?